src/common/btree.c

```diff
@@ -3846,8 +3927,8 @@ btr_class_init(umem_off_t root_off, struct btr_root *root,
 	if (tc->tc_feats & BTR_FEAT_DYNAMIC_ROOT)
 		*tree_feats |= BTR_FEAT_DYNAMIC_ROOT;
 
-	if (tc->tc_feats & BTR_FEAT_SKIP_LEAF_REBAL)
-		*tree_feats |= BTR_FEAT_SKIP_LEAF_REBAL;
+	if (tc->tc_feats & BTR_FEAT_LAZY_LEAF_REBAL)
+		*tree_feats |= BTR_FEAT_LAZY_LEAF_REBAL;
```

Comment:
If only supported for DRAM tree, should we check that here and return an error?

Reply:
It is not important whether it is DRAM or PMEM based, instead, the caller needs to guarantee that there is no CPU yield during the btree operations before rebalance done.

Reply:
What happens if there is a crash before the rebalance is done?

Reply:
If it is a DRAM based btree, then it will recover after server restart. If it is PMEM based, then related transaction will fail and be aborted, then the btree will be recovered.

Reply:
So the dram list of entries is only active during a pmdk transaction?